pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -249,7 +249,11 @@ export default class WindowsIntegrationManager implements IntegrationManager {
 
     await Promise.all(
       (await this.supportedDistros).map((distro) => {
-        return this.syncDistroSocketProxy(distro.name, !reason);
+        try {
+          return this.syncDistroSocketProxy(distro.name, !reason);
+        } catch (error) {
+          console.error(`syncSocketProxy ${ distro.name } Error: ${ error }`);
```

Comment:
`syncDistroSocketProxy` is async, but this anonymous function is not; so the `catch` will not catch rejected promises, but the outer `await Promise.all` will (and still throw an exception).

Given the other changes, do you want to wrap `syncDistroSocketProxy` in `try`/`catch` instead?

Reply:
Nice catch!  Change has been checked in.